### PR TITLE
Rails 6 migration

### PR DESCRIPTION
**Pull request by rolo9128**

### CODE REVIEW

File: maglev.gemspec [hunk 1]
```diff
@@ -26,14 +26,14 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'globalid', '<= 0.4.1'
-  spec.add_dependency 'activemodel', '~> 5.2.0'
+  spec.add_dependency 'globalid', '<= 0.4.2'
```
Comment (jhoffner) [inline maglev.gemspec:29]: We might have to force this back to `0.4.1`. I remember 0.4.2 causing issues with Mongoidal

**Review by jhoffner — CHANGES_REQUESTED** [review]
Please bump the version to 0.5 and update `changed.md`

File: maglev.gemspec (hunk 1 above)
Comment (rolo9128) [reply maglev.gemspec]: Done.

**Comment by jhoffner** [comment]
@rolo9128 looks good, just need to bump the version and update `changes.md`

**Comment by rolo9128** [comment]
@jhoffner good to go now

**Pull request merged**